instrumentation/go.mongodb.org/mongo-driver/mongo/otelmongo/config.go

```diff
@@ -55,3 +57,11 @@ func WithTracerProvider(provider trace.TracerProvider) Option {
 		cfg.TracerProvider = provider
 	}
 }
+
+// WithCommandLoggingDisabled specifies if tracing of the mongo command is disabled or not.
+// If none is specified, logging of the mongo command is enabled.
+func WithCommandLoggingDisabled(disabled bool) Option {
```

Comment:
I'm against using the term `Logging` here. This term will describe the logging signal that we plan to add to the project at a later date.

```suggestion
func WithCommandAttributeDisabled(disabled bool) Option {
```